### PR TITLE
feat: convert the fakePriceAuthority to a PlayerPiano model

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
It repeatedly runs through the list of prices provided at setup. This
will be used to create a price oracle for the test net.

tests updated

### CODE REVIEW

File: packages/ERTP/src/localAmountMath.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { E } from '@agoric/eventual-send';
 import { makeAmountMath } from './amountMath';
 
 /**
- * @param {Issuer} issuer
+ * @param {ERef<Issuer>} issuer
```
Comment (katelynsills) [inline packages/ERTP/src/localAmountMath.js:5]: oh thank you! I kept hitting this

File: packages/zoe/tools/types.js [hunk 2]
```diff
@@ -77,7 +77,11 @@
  * brandIn/brandOut pair
  *
  * @property {(brandIn: Brand, brandOut: Brand) => ERef<Notifier<PriceQuote>>}
- * getPriceNotifier
+ * getPriceNotifier be notified of the latest PriceQuotes for a given
+ * brandIn/brandOut pair.  Note that these are raw quotes, and not for a
+ * standardized amountIn or amountOut.  The fact that they are not scaled means
+ * that a PriceAuthority can implement quotes for both fungible and non-fungible
+ * brands.
```
Comment (katelynsills) [inline packages/zoe/tools/types.js]: I think this is slightly confusing still. For the priceAggregator, we assume that the priceNotifier is giving prices per a standard unit, so saying that this is not standardized seems wrong. Maybe we mean "does not necessarily have to be standardized, but is for the priceAggregator"

**Review by katelynsills — APPROVED** [review]
Looks good, one comment that I think needs to be changed

File: packages/zoe/tools/types.js (hunk 2 above)
Comment (michaelfig) [reply packages/zoe/tools/types.js]: The `priceAggregator` generates quotes with differing `amountIn`s when somebody asks for a `quoteWanted`.  But I have clarified the comment without needing to name the aggregator.

Comment (katelynsills) [reply packages/zoe/tools/types.js]: Right, but the priceNotifier acts differently than `quoteWanted`, right?

**Pull request merged**